devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/resteasy-jackson-example/kotlin/src/main/kotlin/org/acme/resteasyjackson/MyObjectMapperCustomizer.kt

```diff
@@ -6,8 +6,9 @@ import io.quarkus.jackson.ObjectMapperCustomizer
 import javax.inject.Singleton
 
 @Singleton
-class FruitObjectMapperCustomizer : ObjectMapperCustomizer {
+class MyObjectMapperCustomizer : ObjectMapperCustomizer {
```

Comment:
Why use `MyObject` instead of `Fruit` ? `MyObject` is non-descriptive but applies to lots of stuff thus not the greatest to put into an example; as opposed to `Fruit` which is pretty specific and if you as new coder would derive from the example would be naturally inclined to rename ...where as MyObject would just hang out forever (not good)

Reply:
@maxandersen 
- this is not dealing with `Fruit` anymore.
- the mapper behavior is global, it suppresses null value for the output json for all the different resources.

So IMHO `MyObjectMapperCustomizer` is the best possible name, `SuppressNullValueObjectMapperCustomizer` seems a bit overkill and would not allow other global customization. We could also settle on `GlobalObjectMapperCustomize`?

Reply:
ah - thanks for the explantation.  makes sense.  

I guess in "real world" it would be called `AcmeObjectMapperCustomizer` but you support changing packages names now right ? then Acme wouldn't be the consistent name.

This customizer is a specific to Kotlin example? so could be `KotlinObjectMapperCustomizer` ...but I'm probably fine with `MyObjectMapperCustomizer` as is.